packages/account/src/Styles/account.scss

```diff
@@ -1450,10 +1450,10 @@ $MIN_HEIGHT_FLOATING: calc(
                     }
                     &--small {
                         max-width: 99px;
-                        
+
                         .data-list__row-content .dc-btn {
                             left: 50%;
-                            transform: translateX(-52%);
+                            transform: translateX(-53%);
```

Comment:
- This seems like a temporary fix for this issue, better resolve this issue properly.
- Seems like `data-list__item` container does not include all the content inside it and long text can overflow out of the container.
I suggest to discuss this further with design and usability team for a better UI/UX, looks like there is not enough space for button with long text in the design specially for screen sizes such as iPhone 5/SE.